### ISSUE TITLE
Can't expand glob

### ISSUE THREAD

**Issue by Aloxaf** [Aloxaf/fzf-tab#25]

**Comment by romkatv** [comment]
Thanks for opening this issue! Just +1 for me. I'd really like to be able to expand globs liked I used to.

**Comment by Aloxaf** [comment]
Fixed it with a hacky way

**Comment by romkatv** [comment]
Works!

I spent half an hour tracking some awful side effects of this bug when using a combination of `complete-with-dots`, `zsh-syntax-highlighting` and `zsh-autosuggestions`. Basically, when fzf-tab calls the original completion widget, it may have been wrapped by plugins. These plugins may attempt to change highlighting and call `zle -R`. This breaks if the cursor is temporarily in a place unknown to zle. I managed to work around this by manually specifying what gets wrapped and what doesn't. It's hacky but OK with me.
